seahub/api2/endpoints/account.py

```diff
@@ -196,7 +196,7 @@ def put(self, request, email, format=None):
                 space_quota_mb = int(space_quota_mb)
             except ValueError:
                 return api_error(status.HTTP_400_BAD_REQUEST,
-                        'storage invalid.')
+                        _('Must be an integer that greater than or equal to 0.'))
```

Comment:
=> Must be an integer that is greater than or equal to 0.

or

=> Must be an integer being greater than or equal to 0.